subsys/testsuite/ztest/src/ztest_new.c

```diff
@@ -464,12 +476,51 @@ struct ztest_unit_test *ztest_get_next_test(const char *suite, struct ztest_unit
 	return NULL;
 }
 
+#ifdef CONFIG_ZTEST_DO_THE_SHUFFLE
+static void ztest_seed_the_shuffle(void)
+{
+	uint32_t z_seed = SHUFFLE_SEED;
+
+	if (z_seed == 0) {
+		if (IS_ENABLED(CONFIG_POSIX_CLOCK)) {
+			time_t t = 0;
+			z_seed = (uint32_t)time(&t);
+		} else {
+			z_seed = k_cycle_get_32();
+		}
```

Comment:
Can you try `sys_clock_tick_get_32()`?

Reply:
@asemjonovs I figured it out. You can do

```
#include <time.h>

time_t now = time(NULL);
```

`time_t` is a `long int` that you can pass to the seed.